apollo-router/src/plugins/rhai/engine.rs

```diff
@@ -218,6 +219,40 @@ mod router_method {
     }
 }
 
+#[export_module]
+mod status_code {
+    use rhai::INT;
+
+    pub(crate) type StatusCode = http::StatusCode;
+
+    #[rhai_fn(name = "status_code_from_int", return_raw)]
```

Comment:
NIT: Not renaming, so this is basically
```suggestion
    #[rhai_fn(return_raw)]
```